styles/style_ptr.h

```diff
@@ -164,6 +183,18 @@ StyleAllocator StylePtr(const char* default_arguments) {
 // that you can't turn it on/off, and the battery low warning is disabled.
 template<class STYLE>
 StyleAllocator ChargingStylePtr() {
+  using _TopBaseT  = typename style_base_check_detail::TopBase<STYLE>::type;
+  using _TopBaseCol = decltype(std::declval<_TopBaseT&>().getColor(0));
+  static_assert(color_details::IsOpaqueColor<_TopBaseCol>::value,
```

Comment:
These error messages are overkill.
A few stars at the beginning/end should be enough.

Reply:
yeah - calmed them down